src/win/win.c

```diff
@@ -764,7 +762,6 @@ plat_vidapi_enable(int enable)
     if (!vid_api_inited || !vid_apis[vid_api].enable)
 	return;
 
-    video_wait_for_blit();
```

Comment:
This was the most insidious one, because it was the once called during WM_SIZE, and thus the easiest one to reproduce the issue.